### PR TITLE
Metrics2

### PR DESCRIPTION
**Pull request by bxrgem**
I have implemented some metrics for the aggregator. The metrics collected at the moment is total duration of cachedb access, duration of sending to kvalobs and total duration of running computation triged by a single observation.

It also logged if an observation has generated one or more aggregated values that is sendt to kvalobs.

The metric is written to the file /var/log/kvalobs/kvAgregated_metrics.log. 

The aggregator is running on the kvalobs staging host.

### CODE REVIEW

File: src/proxy/KvalobsDataAccess.cpp [hunk 1]
```diff
@@ -88,14 +92,18 @@ void KvalobsDataAccess::getAllData(KvDataList & data, const boost::posix_time::p
     wdh.addStation( station, newFrom, to );
 
     proxy::internal::KvDataReceiver dr( data );
-
-    if ( ! KvApp::kvApp )
+    if ( ! KvApp::kvApp ) {
+			metrics->cacheDb.stop(true);
```
Comment (vegardb) [inline src/proxy/KvalobsDataAccess.cpp]: metrics->kvDB.stop()?

File: src/metrics.h [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    MilliDuration acc_;
+};
+
+
+
+class Metrics {
+  public:
+    Metrics();
+
+    Metric cacheDb;
```
Comment (vegardb) [inline src/metrics.h:63]: A simple comment on what this is would be nice

File: src/metrics.h [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    Metric kvDb;
+
+    int sendtToKvalobs(int isSendt) { sendtToKvalobs_=isSendt;}
```
Comment (vegardb) [inline src/metrics.h]: What is this? why not a bool?

Comment (bxrgem) [reply src/metrics.h]: I started with a bool, but then I remembered  a bug where someone is sending empty data and I woulds test if it was the aggregator, it is not. So I made it a treestate -1 (empty data), 0 (not sendt to kvalobs), 1 (sendt to kvalobs). I will make it a bool.

File: src/proxy/KvalobsDataAccess.cpp (hunk 1 above)
Comment (bxrgem) [reply src/proxy/KvalobsDataAccess.cpp]: This is a bug.

**Review by vegardb — APPROVED** [review]

**Pull request merged**